### PR TITLE
Update lambdas for linting and include in test run

### PR DESCRIPTION
**Pull request by alistairjcbrown**
### What's Changed

Linting tooling has been in place for the Node.js lambdas, but not been used. This PR runs `npm run lint -- --fix` on the lambda JavaScript codebase (as well as some other small code adjustments and linting fixes), and also adds the linting task to the overall codebase test run to fail builds which don't conform to linting standards.

### CODE REVIEW

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -109,6 +109,17 @@ services:
...
       - './services/client:/usr/src/app'
 
+  lambdas-test:
+    container_name: lambdas-test
+    build:
+      context: ./lambdas
+      dockerfile: Dockerfile-test
+    environment:
+      - NODE_ENV=development
+      - CI=${CI:-false}
```
Comment (apoclyps) [inline docker-compose-dev.yml]: Is the CI variable required to run linting on CI?

File: lambdas/Dockerfile-test [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM node:8.10-alpine
```
Comment (apoclyps) [inline lambdas/Dockerfile-test]: Does lambda require this specific version of node?

File: docker-compose-dev.yml (hunk 1 above)
Comment (alistairjcbrown) [reply docker-compose-dev.yml]: @apoclyps Probably not (copy/paste from the client test run), but I'll leave it in place if that's ok to cover when this image is also used for the test run

File: lambdas/Dockerfile-test (hunk 2 above)
Comment (alistairjcbrown) [reply lambdas/Dockerfile-test]: @apoclyps That's the version they're currently running on - https://github.com/apoclyps/my-dev-space/blob/9e87a0166790fde0ccb648c442e3b60eaf7dcdb2/lambdas/meetupcom/serverless.yml#L7

**Pull request merged**